.github/ISSUE_TEMPLATE.md

```diff
@@ -1,35 +1,41 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
```

Comment:
I think that this is too aggressive. Github already links it when you create an issue.

![screen shot 2016-09-15 at 12 57 52 pm](https://cloud.githubusercontent.com/assets/1444314/18565283/0b001cc2-7b44-11e6-8298-4ce8e95bb13c.png)